Simplenote/Classes/UIGestureRecognizer+Simplenote.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import UIKit
+
+
+// MARK: - UIGestureRecognizer Simplenote Methods
+//
+extension UIGestureRecognizer {
+
+    /// Forgive me: this is the only known way (AFAIK) to force a recognizer to fail. Seen in WWDC 2014 (somewhere), and a better way is yet to be found.
+    ///
+    func fail() {
+        isEnabled = false
+        isEnabled = true
```

Comment:
**TIL**
I didn't know this!

Reply:
I'm actually quite ashamed!!! BUT i've seen this at WWDC 2014. So... oh well!!